python/katana/property_graph.pyx.jinja

```diff
@@ -28,27 +31,32 @@ cdef class PropertyGraph:
     """
     A property graph loaded into memory.
     """
-    def __init__(self, path, node_properties=None, edge_properties=None):
+    def __init__(self, path, partition_to_load=None, node_properties=None, edge_properties=None):
         """
-        __init__(self, path, node_properties=None, edge_properties=None)
+        __init__(self, path, partition_to_load=None, node_properties=None, edge_properties=None)
 
         Load a property graph.
 
         :param path: the path or URL from which to load the graph. This support local paths or s3 URLs.
         :type path: str
+        :param partition_to_load: An unsigned integer corresponding to a partition number in the graph being loaded. If this is None (default), then the partition corresponding to this host's network id will be loaded.
```

Comment:
Python doesn't have unsigned ints natively. Just say "The number of the partition to be loaded from the the graph" or similar. The exact type is totally unimportant for this.